qaframework-bdd-tests/src/test/resources/features/refapp-2.x/016-XssVulnerability/xssVulnerability.feature

```diff
@@ -0,0 +1,24 @@
+Feature: Check Xss Vulnerability
+
+  Background:
+    Given User goes to active visit search page
+   
+  @selenium
+  @xssVulnerability
+  Scenario: Check XSS Vulnerability
+    When User searches for patient using patient identifier
+    And User goes to patient dashboard of last Active visit
+    Then User clicks on show contact link
+    And User clicks on editContact info
+    And User clicks on phone Number edit
+    Then User clears phone number
+    And User enters new phone number with xss check script
+    And User clicks on confirm Edit button
+    Then System returns a message validator
+
+    And User re clears the phone Number input area
```

Comment:
```suggestion
    And User clears the phone Number input area
```

Reply:
The idea is to have clean , readable scenario. It however depends on the fact of functionality